src/main/java/ai/susi/json/JsonLDNode.java

```diff
@@ -188,7 +188,11 @@ public Object getPredicateValue(String key) {
 
     public List<String> getPredicates() {
         ArrayList<String> predicates = new ArrayList<>();
-        this.object.keySet().forEach(key -> {if (key.charAt(0) != '@') predicates.add(key);});
+        this.object.keySet().forEach(key -> {
+            if (key.isEmpty() || key.charAt(0) != '@') {
```

Comment:
This will add emoty keys to the predicates!

Reply:
Yes, this is the intention. It's even a requirement for our project: If a property is defined and its value is empty or a null string, it should be recognized as such. Otherwise the information will be lost. From our perspective is a null string also a string, what is a valid value and thus the attribute should not be ignored.